<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Livre</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="4" column="0">
       <widget class="QLineEdit" name="author"/>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Titre du livre:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="bookTitle"/>
      </item>
      <item row="9" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="0">
         <widget class="QPushButton" name="downChapterButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="delChapterButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="upChapterButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="topChapterButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="addChapterButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="bottomChapterButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="6">
         <widget class="QListWidget" name="chaptersList"/>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Chapitres</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Résumé:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <layout class="QVBoxLayout" name="abstractLayout"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Auteur:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Page</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="2" column="4">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Chapitre :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="pageTitle"/>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Texte de la page :</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <layout class="QGridLayout" name="sequelsLayout">
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="sequelText"/>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="delSequelButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="sequelsSize"/>
          </item>
          <item row="1" column="1">
           <widget class="QFontComboBox" name="sequelsFont"/>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="sequelTopButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QToolButton" name="addSequelButton">
            <property name="text">
             <string/>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="sequelDown">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="sequelBottomButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="sequelup">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="sequelsFontButton">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="4">
         <widget class="QComboBox" name="chapterComboBox"/>
        </item>
        <item row="5" column="4">
         <layout class="QVBoxLayout" name="textLayout"/>
        </item>
        <item row="6" column="4">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Liens :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Titre de la page :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Prévisualisation:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
